### ISSUE TITLE
[BUG] tardis changes dims on its own

### ISSUE THREAD

**Issue by AlienXtream** [DarkholmeTenk/TardisModIssues#287]
this bug appears to occur randomly and i know its been around for a while BUT... i saw the lever move when i pressed the day/night button :/

**Comment by DarkholmeTenk** [comment]
Yes, it's caused by the levelling up unlocking a new dimension and when you press a button, it just forces the client to get the new value :P

**Comment by AlienXtream** [comment]
does the tardis not store its loc data in the console TE? unless, oh! are you making a list of unlocked dims and putting them in numerical order? in that case say i have dims 1, 3, 4 unlocked they get assigned to positions 1, 2, 3 respectively but adding dim 2 to the list moves them out of the existing order? sorry, that got a bit confusing didn't it :/

**Comment by DarkholmeTenk** [comment]
You're almost exactly right, I think :) still a bug tho
